bot/cogs/halloweenify.py

```diff
@@ -33,8 +33,9 @@ def __init__(self, bot):
         embed.colour = discord.Colour.dark_orange()
         embed.title = 'Wow!'
```

Comment:
The title could probably be improved a bit, too. Maybe in staying with the theme below, it could be something like `Not spooky enough?`